helper.sh

```diff
@@ -35,7 +35,7 @@ init_boot ()
     touch "${DIR}"/secrets/acme.json # to avoid making it think it's a folder
     chmod 600 "${DIR}"/secrets/acme.json # Traefik doesn't let ACME challenge go through otherwise
 
```

Comment:
Can you add the `touch` I mentioned? so we avoid having a folder

Reply:
is there a point in `touch` ing the file and then overriding it with the correct one?

Reply:
it's not supposed to be not presented

Reply:
it is, because we're talking about different files apparently :) oops